app/templates/fileList.js

```diff
@@ -0,0 +1,29 @@
+const html = require('choo/html');
+const file = require('./file');
+
+module.exports = function(state, emit) {
+  const table = html`
+  <table id="uploaded-files">
+    <thead>
+      <tr>
+        <!-- htmllint attr-bans="false" -->
+        <th id="uploaded-file">${state.translate('uploadedFile')}</th>
+        <th id="copy-file-list">${state.translate('copyFileList')}</th>
+        <th id="expiry-file-list">${state.translate('expiryFileList')}</th>
+        <th id="delete-file-list">${state.translate('deleteFileList')}</th>
+        <!-- htmllint tag-bans="$previous" -->
+      </tr>
+    </thead>
+    <tbody>
+      ${state.storage.files.map(f => file(f, state, emit))}
+    </tbody>
+  </table>
+  `;
+  const list = state.storage.files.length ? table : '';
```

Comment:
Should we check `state.storage.files.length` sooner and just abort early if there are no files? May save some l20n lookups and templating, etc.